package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adhocteam/kongfig",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
updated to a new minor point version because this PR adds a new executable command